apps/studio/.gitignore

```diff
@@ -41,7 +41,7 @@ yarn-error.log*
 playwright/test-results
 
 # env
-.env
+.env*
```

Comment:
what about .env.example? we using env-example ah?

Reply:
will make it less restrictive first